### DISCUSSION TITLE
Unable to find a workflow for class "Pimcore\Model\DataObject\CoreShopOrder coreshop 4 with pimcore 11

### DISCUSSION THREAD

**Discussion by claudesimo1990** [coreshop/CoreShop#2649] [General]
I get this error when I want to complete the order process, does anyone have an idea why I get this?

![image](https://github.com/coreshop/CoreShop/assets/45970298/5fc9977a-dcb7-4fc0-806f-5f947e9f4955)

**Comment by dpfaffenbauer** [comment]
Hard to tell honestly. Is it a new Project? Is it a Update?

**Reply by claudesimo1990** [reply]
It is a new Project

**Reply by claudesimo1990** [reply]
![image](https://github.com/coreshop/CoreShop/assets/45970298/f047180b-b360-47b4-b6b7-eb408290938b)

it is this workflow: coreshop_order_sales_type

**Reply by dpfaffenbauer** [reply]
Anything you changed already?

**Reply by claudesimo1990** [reply]
No, not at all, it's a brand new project. coreshop version 4.0.5

**Reply by dpfaffenbauer** [reply]
what did you do to set it up? we have so many different tests and demos that something like that should be known

**Reply by claudesimo1990** [reply]
Hi @dominikager  I've just changed the schop theme and inherited the productController to add pagination, but otherwise I really don't see any major changes. I've migrated the version to 4.0.7 but I don't see any changes.

I only get these errors on actions like buy and quote.

https://gitlab.com/code3612546/bookshop

**Reply by dpfaffenbauer** [reply]
@claudesimo1990 It is really hard to understand whats going on. It works on all of our tests, on all of our demos and in all of our projects...

**Reply by claudesimo1990** [reply]
@dpfaffenbauer Good evening, after an update everything is working normally again. Thank you.
